src/components/Popups/PopupReply.tsx

```diff
@@ -1,10 +1,13 @@
 import * as React from 'react';
 import * as Popper from '@popperjs/core';
 import * as ReactRedux from 'react-redux';
+import { useIntl } from 'react-intl';
+import FocusTrap from 'box-ui-elements/es/components/focus-trap/FocusTrap';
 import PopupBase from './PopupBase';
 import ReplyForm from '../ReplyForm';
 import usePrevious from '../../common/usePrevious';
 import { getScale, getRotation } from '../../store/options';
+import messages from '../Popups/messages';
```

Comment:
Can we move this above the `PopupBase` import? Can we shorten it to `import messages from './messages'`?